.github/workflows/default-config.yml

```diff
@@ -6,9 +6,11 @@ on:
   push:
     branches:
       - feature/**
-      - main
+      - master
```

Comment:
@aravindmetku  Does this mean that we didn't have the iAPI default processors correctly deployed into prod workspaces earlier?

Reply:
I think we did. Even I was wondering but it was working for others.